docs/architecture/psa-migration/strategy.md

```diff
@@ -12,11 +12,9 @@ G3. Allow isolation of short-term secrets (for example, TLS session keys).
 G4. Have a clean, unified API for Crypto (retire the legacy API).
 G5. Code size: compile out our implementation when a driver is available.
 
-As of Mbed TLS 3.2, most of (G1) and all of (G2) is implemented when
-`MBEDTLS_USE_PSA_CRYPTO` is enabled. For (G2) to take effect, the application
-needs to be changed to use new APIs. For a more detailed account of what's
-implemented, see `docs/use-psa-crypto.md`, where new APIs are about (G2), and
-internal changes implement (G1).
+As of Mbed TLS 3.2, most of (G1) and all of (G2) is implemented. For a more
+detailed account of what's implemented, see `docs/use-psa-crypto.md`, where new
+APIs are about (G2), and internal changes implement (G1).
```

Comment:
`docs/use-psa-crypto.md` is now being deleted. We are effectively finalizing the transition from “all is legacy-only except what `MBEDTLS_USE_PSA_CRYPTO` supports” to “all is PSA except for a few limitations”. So now I think the relevant reference is `docs/architecture/psa-migration/psa-limitations.md` for how we've only achieved most of (G1) and not all.